rules/EarlyReturn/Rector/If_/ChangeAndIfToEarlyReturnRector.php

```diff
@@ -201,27 +201,27 @@ private function isInLoopWithoutContinueOrBreak(If_ $if): bool
     private function processReplaceIfs(
         If_ $if,
         array $conditions,
-        Return_ $ifNextReturnClone,
+        Return_ $cloneIffNextReturn,
```

Comment:
are these class renaming changes related to the actual goal of this PR?

if not, separating these would help to focus on whats important

Reply:
They're result of fixed naming check :+1:  The endsWith() was buggy.